protocol/triple/server.go

```diff
@@ -409,3 +422,14 @@ func createServiceInfoWithReflection(svc common.RPCService) *server.ServiceInfo
 
 	return &info
 }
+
+// generateAttachments transfer http.Header to map[string]interface{} and make all keys lowercase
+func generateAttachments(header http.Header) map[string]interface{} {
+	attachments := make(map[string]interface{}, len(header))
+	for key, val := range header {
+		lowerKey := strings.ToLower(key)
+		attachments[lowerKey] = val
+	}
+
+	return attachments
+}
```

Comment:
Will there be some HTTP protocol headers here? Do we need to filter them out?

Reply:
Old triple would expose HTTP protocol headers too.

Reply:
ok